black.py

```diff
@@ -541,7 +541,14 @@ def reformat_many(
     if sys.platform == "win32":
         # Work around https://bugs.python.org/issue26903
         worker_count = min(worker_count, 61)
-    executor = ProcessPoolExecutor(max_workers=worker_count)
+    try
```

Comment:
you're missing a `:` here

Reply:
yes sorry you've been faster than me to see I forgot to amend before pushing :)